### PR TITLE
[JBPM-10166] Allow tests with productized Kafka version

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**: [JBPM-10166](https://issues.redhat.com/browse/JBPM-10166)

### CODE REVIEW

**Comment by gmunozfe** [comment]
@fjtirado please review

File: jbpm-workitem-itests/src/test/java/org/jbpm/workitem/springboot/samples/KafkaProxyAsyncSampleTest.java [hunk 1]
```diff
@@ -138,7 +138,9 @@ public void testAsyncKafkaWIHNoConnection() throws Exception {
             
             startOtherProcess();
         } finally {
-            processService.abortProcessInstance(processInstanceId);
+            if (runtimeDataService.getProcessInstances(singletonList(STATE_ACTIVE), null, null).size() == 1) {
```
Comment (fjtirado) [inline jbpm-workitem-itests/src/test/java/org/jbpm/workitem/springboot/samples/KafkaProxyAsyncSampleTest.java:141]: why not invokeind abortPRocess just when the list is not empty
The other option is just wrap the call to abort between a try catch
I personally leave it as it was originally (if the abort calls fails, the test should fail)

**Review by fjtirado — APPROVED** [review]

File: jbpm-workitem-itests/src/test/java/org/jbpm/workitem/springboot/samples/KafkaProxyAsyncSampleTest.java (hunk 1 above)
Comment (gmunozfe) [reply jbpm-workitem-itests/src/test/java/org/jbpm/workitem/springboot/samples/KafkaProxyAsyncSampleTest.java]: thanks, added an abortAllProcesses method as talked

**Comment by gmunozfe** [comment]
@mareknovotny could you merge?

**Pull request merged**